email_handler.py

```diff
@@ -908,6 +925,26 @@ def forward_email_to_mailbox(
                 f"""PGP encryption fails with {mailbox.email}'s PGP key""",
             )
 
+    # create SMIME email if needed
+    if mailbox.smime_enabled() and user.is_premium():
+        LOG.d("Encrypt message using S/MIME for mailbox %s", mailbox)
+
+        try:
+            msg = prepare_smime_message(msg, mailbox.smime_public_key)
+        except Exception as exceptasdf:
+            LOG.w(
+                "Cannot S/MIME encrypt message %s -> %s. %s %s",
+                contact,
+                alias,
+                mailbox,
+                user,
+            )
+            LOG.w(exceptasdf)
```

Comment:
Oops, sorry; I can clean up this variable name and logging

Reply:
👍